### DISCUSSION TITLE
Library entrypoint

### DISCUSSION THREAD

**Discussion by pepoviola** [paritytech/zombienet-sdk#67] [Ideas]
At some point in the roadmap we should decide who we want to expose the library, which will be  the `entrypoint`, what will include and which features will have.

I think make sense to have a `zombienet` crate in the workspace that `act` as main entrypoint, exposing all the needed types to interact with the library. And related to features I think we should leave `logger` / `filesystem` / `test-runner` enabled by default but behind a `feature` flag.

Also, we can publish the `configuration` and  test-runner` independent to allow users to use those in other apps.

What's your feedback @wirednkod / @l0r1s ?

**Comment by jmg-duarte** [comment]
> Also, we can publish the configuration and test-runner` independent to allow users to use those in other apps.

From my experience, until users ask for this, it's more of a burden than anything else
